docs/get-started/getting-started-on-prem.md

```diff
@@ -429,7 +429,7 @@ In order to enable _RapidsShuffleManager_, UCX user-space libraries and its depe
 installed on the host and within Docker containers. A host has additional requirements, like the 
 MLNX_OFED driver and `nv_peer_mem` kernel module.
 
-#### Baremetal
+#### <a name="ucx-baremetal"></a>Baremetal
```

Comment:
Markdown has built-in anchors for headings, no need to generate them manually.  The anchor is the same as the heading words but all lowercase and hyphens between the words.  This one would simply be `#baremetal`.

Reply:
done.